tests/zeus/vcs/conftest.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from uuid import uuid4
+
+from zeus.vcs.server import build_app
+
+
+@pytest.fixture
+def asyncdb(app):
+    return app["db"]
+
+
+@pytest.fixture(scope="function", autouse=True)
+async def asyncdb_transaction(item, asyncdb):
```

Comment:
ideally this only does autouse when the given test has the asyncio marker, but i didnt get far enough to test that yet

Reply:
my original idea wont work, so this will execute always under vcs/*